pkg/cloud/cloud.go

```diff
@@ -299,6 +308,9 @@ func (c *cloud) CreateDisk(ctx context.Context, volumeName string, diskOptions *
 	if iops > 0 {
 		request.Iops = aws.Int64(iops)
 	}
+	if throughput > 0 {
```

Comment:
Can you change this to `throughput > 0 && diskOptions.VolumeType == VolumeTypeGP3`? Let's play it safe as throughput is a valid parameter only for gp3.